<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldsForm</class>
 <widget class="QWidget" name="FieldsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fields</string>
  </property>
  <widget class="QComboBox" name="comboBoxFromSys">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Jira</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bitbucket</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="titleSettings_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Fields</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelFromSys">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>from</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelToSysVar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bitbucket</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFromPath">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxToPath">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveChanges">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>save changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveAsNewMapping">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>save as new field mapping</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDeleteFieldMapping">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddOptionMapping">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>add new option mapping</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>209</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="SaveOptionLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
